about.html

```diff
@@ -0,0 +1,134 @@
+        <a href="#"><i class="uil uil-twitter-alt"></i></a>
+        <a href="#">English</a>
+        <a href="#">Page</a>
+        <a href="#">Logout</a>
+      </div>
+      <div class="hamburger">
+        <button id="hamburger"><i class="uil uil-bars"></i></button>
+        <div class="nav-menu">
+          <div class="desktop-logo">
+            <a href="./index.html"
+              ><img src="./images/white-logo1.jpg" alt="live-concerts-logo"
+            /></a>
+          </div>
+          <div class="menu-list">
+            <img
+              class="close"
+              src="./images/menu-cross.png"
+              alt="close button"
+            />
+            <ul class="about-menu">
+              <li class="close"><a href="./about.html">About</a></li>
+              <li class="close"><a href="#events">Events</a></li>
+              <li class="close"><a href="#">Join</a></li>
+              <li class="close"><a href="#">Sponsor</a></li>
+              <li class="close"><a href="#">News</a></li>
+              <li class="book-now close"><a href="#"> Book Now</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <main class="about-main">
+        <div class="about-title">
+          <h2>"South Africa's best marketplace for live events"</h2>
+          <h1>LIVE CONCERTS MUSIC AT THE LAKE 2022</h1>
+          <p class="about-para1">
+            This is the event never to be missed, enjoy your favourite music
+            with a lovely view while watching a beautiful sunset Lorem ipsum,
+            dolor sit amet consectetur adipisicing elit. Natus, doloremque enim!
+            Necessitatibus fuga voluptas voluptatibus. Quia, laboriosam deleniti
+            omnis nam aliquam libero accusantium dolorum dolor corrupti placeat
+            tempore, nesciunt debitis.
+          </p>
+          <p class="about-para2">
+            For further details about music at the lake 2022, please contact us
+            by email
+          </p>
+          <p class="para2-email">musicatthelake@liveconcerts.co.za</p>
+        </div>
+      </main>
+    </section>
+    <section class="LC-logo">
+      <h2>Live Concerts Music At The Lake Logo 2022</h2>
+      <p>The logo for this event was made by Siphosethu Mahlangu</p>
+      <hr class="breakline" />
+      <img src="./images/white-logo1.jpg" alt="live-concerts-logo" />
+    </section>
+    <section class="about-events">
+      <div class="past-events">
+        <h2>See our past Live Events</h2>
+        <hr class="breakline" />
+        <p>
+          Our last two live events which took place in Durban and Cape Town
+          South Africa
+        </p>
+        <img
+          class="img1"
+          src="./images/event-y2.jpg"
+          alt="Picture of singing group"
+        />
+        <img
+          class="img2"
+          src="./images/past-events2.jpg"
+          alt="Picture of live concert"
+        />
```

Comment:
- [ ] These images should have an orange blend like in the original design.
### Your design
![image](https://user-images.githubusercontent.com/45460192/198296809-1787eb8d-69e3-4025-8600-dc49e9d8b7a8.png)
### Required 
![image](https://user-images.githubusercontent.com/45460192/198296902-30d48072-2a14-4811-924d-524f065e8a60.png)